crates/fj-export/src/lib.rs

```diff
@@ -121,3 +111,27 @@ fn export_stl(mesh: &Mesh<Point<3>>, path: &Path) -> Result<()> {
 
     Ok(())
 }
+
+/// An error that can occur while exporting
+#[derive(Debug, Error)]
+pub enum Error {
+    /// No extension specified
+    #[error("no extension specified")]
+    NoExtension,
+
+    /// Unrecognised extension found `{0:?}`
```

Comment:
Copy-pasting the error message works for the other doc comments, but not this one :grin: 
```suggestion
    /// Unrecognised extension found
```